### PR TITLE
update release note

### PR DESCRIPTION
**Pull request by Binyang2014**
Update release note
Remove RELEASE_NOTE_zh_CN.md

### CODE REVIEW

File: RELEASE_NOTE.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+## July 2020 (version 1.1.0)
+
+- Storage:
+  - Support readonly storage. (#4523)
+- Security
+  - If ssl is enabled, all request in webportal will use https. (#4550)
+- Authentication
+  - Support nested AD group in AAD Mode. (#4639)
+- Marketplace
+  - Integral with new version of [PAI marketplace](https://github.com/microsoft/openpaimarketplace).
```
Comment (debuggy) [inline RELEASE_NOTE.md:12]: Integrate

Comment (debuggy) [inline RELEASE_NOTE.md:12]: Integrate

File: RELEASE_NOTE.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
 # OpenPAI Release Note
 
+## July 2020 (version 1.1.0)
+
+- Storage:
+  - Support readonly storage. (#4523)
+- Security
+  - If ssl is enabled, all request in webportal will use https. (#4550)
```
Comment (abuccts) [inline RELEASE_NOTE.md:8]: not only for webportal

Comment (abuccts) [inline RELEASE_NOTE.md:8]: all requests

**Pull request merged**